create-cert-bundle.py

```diff
@@ -0,0 +1,50 @@
+import os
+import logging
+import sys
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+proxy_url = os.environ.get("ALL_PROXY")
+if proxy_url is None:
+    # No proxy configured
+    sys.exit(0)
+
+# Just try to check if proxy scheme is http or https
+# This does not validate the full url format
+scheme = proxy_url.split(":")[0]
+if scheme != "https":
+    # Not https proxy
+    sys.exit(0)
+
+logger.info("Creating certificate bundle with proxy root CA, and installing dependencies")
+try:
+    import certifi
+
+except ImportError:
+    try:
+        from pip._vendor import certifi
+    except Exception as e:
+        logger.exception("Error creating certificate bundle with proxy root CA")
```

Comment:
can we be more specific here? what does the customer need to do here, e.g. `pip install certifi`? should you log the exception?

Reply:
I don't know how to proceed in that case, as we need the cert bundle for pip install itself. Should we just give up?